<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ПВО: Детектор шариков</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon.ico</normaloff>:/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>63</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Звуковуха</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbSoundcardList"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnStartup">
          <property name="maximumSize">
           <size>
            <width>74</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Пуск</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Decoded OK</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="4">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="6">
            <widget class="QLineEdit" name="tbAltitude"/>
           </item>
           <item row="0" column="5">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>YMARS</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>SATS:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="lblSats">
             <property name="text">
              <string>?</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3" colspan="4">
            <widget class="QLabel" name="lblWX">
             <property name="text">
              <string>?</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Altitude</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Weather:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>GOOGLE</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="tbPosition">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="5" column="3" colspan="2">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>160000</height>
              </size>
             </property>
             <property name="title">
              <string>Target</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_6">
                <property name="leftMargin">
                 <number>8</number>
                </property>
                <property name="topMargin">
                 <number>2</number>
                </property>
                <property name="rightMargin">
                 <number>8</number>
                </property>
                <property name="bottomMargin">
                 <number>2</number>
                </property>
                <property name="verticalSpacing">
                 <number>5</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Heading</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Distance</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="lb_tgt_heading">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="lb_tgt_distance">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Position</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>83</height>
              </size>
             </property>
             <property name="title">
              <string>My Position</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="1" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Latitude</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="ed_my_long">
                  <property name="text">
                   <string>43.0</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Longitude</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="ed_my_lat">
                  <property name="text">
                   <string>56.0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="5" column="5">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="cb_gotLock">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>LOCK</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_writeLog">
               <property name="text">
                <string>Write Log</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbInvert">
               <property name="text">
                <string>Invert signal</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="6">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="btn_gpc_window">
               <property name="text">
                <string>GPS Setup</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>GPS Enable</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QProgressBar" name="pbInputLevel">
               <property name="maximum">
                <number>32767</number>
               </property>
               <property name="value">
                <number>24</number>
               </property>
               <property name="textVisible">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>75</height>
         </size>
        </property>
        <property name="title">
         <string>RAW Data</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTextEdit" name="rawDataDisplay">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;RAW DATA: Любые принятые пакеты, необязательно валидные.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
